test_utilities/bin/global_test_runner.dart

```diff
@@ -23,20 +22,24 @@ Future<Null> main(List<String> rawArgs) async {
   var doc = loadYaml(file.readAsStringSync());
   // Execute the tests
   String baseDir = normalize(join(dirname(Platform.script.toFilePath()), '..', '..'));
-  doc['tasks'].forEach((task){
+  String prepareScriptPath = join(baseDir, 'test_utilities', 'bin', 'prepare_environment.sh');
+  await runShellCommand(<String>[prepareScriptPath], 'prepare environment');
+  doc['tasks'].forEach((task) async {
     String scriptPath = join(baseDir, task['script']);
     String taskPath = join(baseDir, task['task']);
-    Process.run('sh', <String>[scriptPath, taskPath]).then((result) {
+    await runShellCommand(<String>[scriptPath, taskPath], task['task']);
+  });
+}
+
+void runShellCommand(List<String> args, String taskName) async {
+  Process.run('sh', args).then((result) {
```

Comment:
Nit: I'm slightly concerned about using `sh` might not be consistent across different platforms.

Reply:
Cocoon tests are currently running only on Linux.